docs/guides/developer/docs/architecture/api/playlists-api.md

```diff
@@ -0,0 +1,225 @@
+| `sort`                   | [`string`](types.md#basic) | Sort the results based upon a sorting criteria. A criteria is specified as a pair such as: <Sort Name>:ASC or <Sort Name>:DESC. Adding the suffix ASC or DESC sets the order as ascending or descending order and is mandatory. Sort Name is case sensitive. Supported Sort Names are 'updated' |
+
+__Example__
+
+```json
+{
+  "playlist": [
+    {
+      "playlist": {
+        "id": 551,
+        "organization": "mh_default_org",
+        "entries": [
+          {
+            "id": 553,
+            "contentId": "ID-about-opencast",
+            "type": "EVENT"
+          },
+          {
+            "id": 554,
+            "contentId": "ID-3d-print",
+            "type": "EVENT"
+          }
+        ],
+        "title": "Opencast Playlist",
+        "description": "This is a playlist about Opencast",
+        "creator": "Opencast",
+        "updated": 1701854481056,
+        "accessControlEntries": {
+          "allow": true,
+          "role": "ROLE_USER_BOB",
+          "action": "read"
+        }
+      }
+    },
+    {
+      "playlist": {
+        "id": 1051,
+        "organization": "mh_default_org",
+        "entries": [
+          {
+            "id": 1053,
+            "contentId": "ID-about-opencast",
+            "type": "EVENT"
+          },
+          {
+            "id": 1054,
+            "contentId": "ID-3d-print",
+            "type": "EVENT"
+          }
+        ],
+        "title": "Opencast Playlist",
+        "description": "This is a playlist about Opencast",
+        "creator": "Opencast",
+        "updated": 1701856455007,
+        "accessControlEntries": {
+          "allow": true,
+          "role": "ROLE_USER_BOB",
+          "action": "read"
+        }
+      }
+    }
+  ]
+}
+```
+
+### PUT /api/playlists/update.json
+
+Updates a playlist or creates a new one.
+
+__Response__
+
+`200 (OK)`: The updated playlist.
+`400 (BAD REQUEST)`: The request is invalid or inconsistent.  
+`403 (FORBIDDEN)`: The user doesn't have the rights to make this request.  
+
+| Field       | Type                            | Description             |
+|-------------|---------------------------------|-------------------------|
+| `playlist`  | [`Playlist`](types.md#Playlist) | Playlist in JSON format |
+
+
+__Example__
+
+```json
+{
+  "playlist": {
+    "id": 551,
+    "organization": "mh_default_org",
+    "entries": [
+      {
+        "id": 553,
+        "contentId": "ID-about-opencast",
+        "type": "EVENT"
+      },
+      {
+        "id": 554,
+        "contentId": "ID-3d-print",
+        "type": "EVENT"
+      }
+    ],
+    "title": "Opencast Playlist",
+    "description": "This is a playlist about Opencast",
+    "creator": "Opencast",
+    "updated": 1701854481056,
+    "accessControlEntries": {
```

Comment:
I haven't looked into the code. Shouldn't `accessControlEntries` be an array instead?

Example: 
```
"accessControlEntries": [
   {
      "allow": true,
      "role": "ROLE_USER_BOB",
      "action": "read"
    }
],
```

Same question for the other API paths.